.ko.yaml

```diff
@@ -2,3 +2,8 @@ defaultBaseImage: public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-bas
 defaultPlatforms:
   - linux/arm64
   - linux/amd64
+builds: 
```

Comment:
We don't want to build it like this always, right? Only in testing? How do we ensure that this only gets added when we are building a testing snapshot?

Reply:
We may want to actually build a completely separate image from the ones that we vend out to customers for this kind of thing